test/ngAnimate/animateSpec.js

```diff
@@ -1543,6 +1543,49 @@ describe("animations", function() {
       });
     });
 
+    they('should not animate an element when the pinned element (which is the $prop element), is pinned to an element that is not a child of the $rootElement',
```

Comment:
Really? Shouldn't jshint / jscs report this then? I thought we were at 200 chars.